.github/workflows/tests-smoke-ipv6.yaml

```
@@ -88,6 +88,7 @@ jobs:
         run: |
           until curl --silent -f -lSL "https://quay.io/api/v1/repository/${{ github.repository_owner }}/cilium-ci/tag/${{ steps.vars.outputs.tag }}/images" &> /dev/null; do sleep 45s; done
           until curl --silent -f -lSL "https://quay.io/api/v1/repository/${{ github.repository_owner }}/operator-generic-ci/tag/${{ steps.vars.outputs.tag }}/images" &> /dev/null; do sleep 45s; done
+          until curl --silent -f -lSL "https://quay.io/api/v1/repository/${{ github.repository_owner }}/hubble-relay-ci/tag/${{ steps.vars.outputs.tag }}/images" &> /dev/null; do sleep 45s; done
```

Since we are starting to get many lines, I suggest we replace all 3 lines with:
```suggestion
          for image in cilium-ci operator-generic-ci hubble-relay-ci ; do
            until curl --silent -f -lSL "https://quay.io/api/v1/repository/${{ github.repository_owner }}/$image/tag/${{ steps.vars.outputs.tag }}/images" &> /dev/null; do sleep 45s; done
          done
```
which is also what we do in conformance workflows :)

---

Done

---

Thanks, though I think it will not work in this state because there's only one `done` for the `until`, and we're missing a second `done` for the `for` loop 😄